api/kyverno/v1/policy_types.go

```diff
@@ -623,3 +629,9 @@ type ResourceSpec struct {
 	// Name specifies the resource name.
 	Name string `json:"name,omitempty" yaml:"name,omitempty"`
 }
+
+type ValidationFailureActionOverride struct {
+	// +kubebuilder:validation:Enum=audit;enforce
+	Action     string   `json:"action,omitempty" yaml:"action,omitempty"`
```

Comment:
So the only two allowed values are either `audit` or `enforce`. I think it would be smart to define two variable constants with `ActionAudit = audit` and `ActionEnforce = enforce`. That we do not need to handle magic strings in multiple places in our code but have one reference for it.

See as an example: https://github.com/kubernetes-sigs/cluster-api/blob/6b042456383aa18ba61fc5de5c41901dd7267c60/test/infrastructure/docker/api/v1beta1/dockercluster_types.go#L25-L29

Reply:
This tag `// +kubebuilder:validation:Enum=audit;enforce` helps validate `Action` while operating policies, and we have these two defined in https://github.com/kyverno/kyverno/blob/25722366f0777dd7306693c8d085c654b4390470/pkg/common/common.go#L24-L29.

Just reviewing the PR, @ojhaarjun1 - looks like you already found that!